Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -10529,6 +10534,7 @@ def remove_old_incidents_ids(found_incidents_ids, current_time, look_back):
     :return: The new incidents ids
     :rtype: ``dict``
     """
+    demisto.debug('Remove old incidents ids')
```

Comment:
where does the removal occur in this method?
also I wouldn't print this unless it contains a variable value